src/routes/ViewJobLog.js

```diff
@@ -5,11 +5,11 @@ import { FormattedMessage } from 'react-intl';
 import { stripesConnect } from '@folio/stripes/core';
 
 import { Preloader } from '../components/Preloader';
-import { LogColorizer } from '../components/CodeHighlight';
```

Comment:
Make sure to delete sources for that component.

Reply:
Done